hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/pages/datanodes/datanodes.tsx

```diff
@@ -294,17 +294,17 @@ const Datanodes: React.FC<{}> = () => {
           onCancel={handleModalCancel}
           closable={false}
           width={400} >
-            <div style={{
-              margin: '0px 0px 5px 0px',
-              fontSize: '16px',
-              fontWeight: 'bold'
-            }}
+        <div style={{
+          margin: '0px 0px 5px 0px',
+          fontSize: '16px',
+          fontWeight: 'bold'
+        }}
           data-testid='dn-remove-modal'>
-              <WarningFilled className='icon-warning' style={{paddingRight: '8px'}}/>
-              Stop Tracking Datanode
+            <WarningFilled className='icon-warning' style={{paddingRight: '8px'}}/>
+            Stop Tracking Datanode
             </div>
             Are you sure, you want recon to stop tracking the selected <strong>{selectedRows.length}</strong> datanode(s)?
-        </Modal>
```

Comment:
Do we need these changes in the patch?
It seems to be just an extra space added for each line.